models/pattern/patterns/application/argo.go

```diff
@@ -0,0 +1,214 @@
+package application
+
+import (
+	"fmt"
+
+	"github.com/layer5io/meshery/models/pattern/patterns/application/argo/v1alpha1"
+	patternUtils "github.com/layer5io/meshery/models/pattern/utils"
+	"github.com/layer5io/meshkit/utils"
+	meshkube "github.com/layer5io/meshkit/utils/kubernetes"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const argoManifest = "https://raw.githubusercontent.com/argoproj/argo-rollouts/stable/manifests/install.yaml"
```

Comment:
Needs to be noted in the air-gapped deployment manual.

Reply:
Reference needs to be externalized to a Meshery Cloud as an artifact repository.

Reply:
Issue opened https://github.com/layer5io/meshery/issues/3085